### ISSUE TITLE
I don't have enough permission or there's a scope problem

### ISSUE THREAD

**Issue by jennybc** [tidyverse/website-analytics#1]
Tried to run `documentation.R` but hit this, even after clicking through various auth screens:

``` r
> topics_raw <- analytics(c("hostname", "pagePath"), dim_filters = filter_docs)
2019-11-12 09:12:09> Request Status Code: 403
Error: API returned: Request had insufficient authentication scopes.
```

**Comment by hadley** [comment]
Can you check that you have access by going to https://analytics.google.com/analytics/web/?authuser=1#/report-home/a115082821w171131299p170811407 ?  (It certainly looks like you have permission in the admin screen)

**Comment by jennybc** [comment]
That lead me to

<img width="463" alt="Screen Shot 2019-11-12 at 9 25 06 AM" src="https://user-images.githubusercontent.com/599454/68694671-6f2bd700-052e-11ea-8066-d59b1e87c8ea.png">

I clicked "Request Access". Which lead me to

<img width="652" alt="Screen Shot 2019-11-12 at 9 25 21 AM" src="https://user-images.githubusercontent.com/599454/68694705-7ce15c80-052e-11ea-9d42-c8febe9529e2.png">

I clicked "Confirm".

So maybe that will yield more info on your end re: what's missing for other folks.

**Comment by jennybc** [comment]
Oh wait, all of that pertained to my regular gmail account. Presumably because of the `?authuser=1` in the URL.

**Comment by jennybc** [comment]
~When I forced that to happen with my rstudio identity, it still said I needed to request some permission, which I did, which was immediately granted. Perhaps because I perversely didn't have the permission, but have the power to grant it to myself?~ *that was all false*

**Comment by jennybc** [comment]
None of this has changed my (lack of) ability to run the script, though.

**Comment by jennybc** [comment]
Correction: when I go to that URL, using my RStudio Google identity, I get this:

<img width="446" alt="Screen Shot 2019-11-12 at 9 51 34 AM" src="https://user-images.githubusercontent.com/599454/68696595-1bbb8800-0532-11ea-9fb1-39eb0932fa28.png">

**Comment by hadley** [comment]
Fixed in 902be53909e0109b4a85457068f532d14c5606ba

**Comment by jennybc** [comment]
Recording for posterity:

The oauth app change was probably a red herring.

We think ec6ea2b2f56803860ed4bf3aa8ef437761b2ee2b is the real fix.
